### PR TITLE
feature : French translation

### PR DESCRIPTION
**Pull request by laurerunser**
Hi !
I added the French translation to strings.xml and ckchangelog.xml
I did my best to follow the instructions in #48 but tell me if I made any mistakes.

If you think I did a good job, please consider adding the hacktoberfest-accepted label to the merged PR.

Thanks !

### CODE REVIEW

File: Umweltzone/src/main/res/values-fr/ckchangelog.xml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (johnjohndoe) [inline Umweltzone/src/main/res/values-fr/ckchangelog.xml:1]: Please add the `Copyright` block as in other language files but with your name.

File: Umweltzone/src/main/res/values-fr/strings.xml [hunk 2]
```diff
@@ -0,0 +1,185 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (johnjohndoe) [inline Umweltzone/src/main/res/values-fr/strings.xml:1]: Please add the `Copyright` block as in other language files but with your name.

File: Umweltzone/src/main/res/values-fr/ckchangelog.xml [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+
+<resources xmlns:xliff="urn:oasis:names:tc:xliff:document:1.2">
+    <string name="changelog_title">Ce qui a changé</string>
+    <string name="changelog_show_full">Voir plus &#8230;</string>
+    <string name="changelog_version_format">
+        Version <xliff:g id="version_name">%s</xliff:g>
+    </string>
+    <string name="changelog_error_resource_not_android_5">"        Erreur lors de l&#8217;affichage des informations sur la version   "</string>
```
Comment (johnjohndoe) [inline Umweltzone/src/main/res/values-fr/ckchangelog.xml:9]: Please remove the wrapping double quotes (`"`) and the unneeded whitespace. These cause formatting issues.

File: Umweltzone/src/main/res/values-fr/strings.xml [hunk 4]
```diff
@@ -0,0 +1,185 @@
...
+    <string name="action_share_app">Partagez cette application&#8230;</string>
+    <string name="action_changelog">Informations sur la version</string>
+    <string name="share_intent_message">
+        "<![CDATA[Essayez Umweltzone - l&#8217;application Android qui a toutes les informations
+        sur les zones à faible pollution et les zones d'interdiction des diesels en Allemagne.
+        \nTéléchargez-la ici gratuitement : https://play.google.com/store/apps/details?id=de.avpptr.umweltzone]]>"</string>
+    <string name="map_my_location_activation_hint">
+        "Cliquez ici pour configurer votre location et afficher votre position sur la carte."</string>
+    <string name="feedback_activity_title">Retours</string>
+    <string name="appinfo_application_logo_description">"Logo de l&#8217;application &#8220;Umweltzone&#8221;"</string>
```
Comment (johnjohndoe) [inline Umweltzone/src/main/res/values-fr/strings.xml:13]: Please remove the wrapping double quotes (`"`).

File: Umweltzone/src/main/res/values-fr/strings.xml [hunk 5]
```diff
@@ -0,0 +1,185 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources xmlns:xliff="urn:oasis:names:tc:xliff:document:1.2">
+    <string name="action_cities">Villes</string>
+    <string name="action_share_app">Partagez cette application&#8230;</string>
+    <string name="action_changelog">Informations sur la version</string>
+    <string name="share_intent_message">
+        "<![CDATA[Essayez Umweltzone - l&#8217;application Android qui a toutes les informations
+        sur les zones à faible pollution et les zones d'interdiction des diesels en Allemagne.
+        \nTéléchargez-la ici gratuitement : https://play.google.com/store/apps/details?id=de.avpptr.umweltzone]]>"</string>
```
Comment (johnjohndoe) [inline Umweltzone/src/main/res/values-fr/strings.xml:9]: Please remove the wrapping double quotes (`"`).

File: Umweltzone/src/main/res/values-fr/strings.xml [hunk 6]
```diff
@@ -0,0 +1,185 @@
...
+    <string name="map_my_location_activation_hint">
+        "Cliquez ici pour configurer votre location et afficher votre position sur la carte."</string>
+    <string name="feedback_activity_title">Retours</string>
+    <string name="appinfo_application_logo_description">"Logo de l&#8217;application &#8220;Umweltzone&#8221;"</string>
+    <string name="about_activity_title">A propos</string>
+    <string name="appinfo_subtitle">
+        Une application Android à propos des zones à faible émissions (Umweltzonen en allemand) et
+        des zones d&#8217;interdiction du diesel en Allemagne.</string>
+    <string name="appinfo_copyright_title">Idée et réalisation</string>
+    <string name="appinfo_copyright_content">"         (&#169;) Tobias Preuss, 2013 &#8211; 2020     "</string>
```
Comment (johnjohndoe) [inline Umweltzone/src/main/res/values-fr/strings.xml:19]: Please remove the wrapping double quotes (`"`) and the unneeded whitespace. These cause formatting issues.

File: Umweltzone/src/main/res/values-fr/strings.xml [hunk 7]
```diff
@@ -0,0 +1,185 @@
...
+        Nous ne pouvons apporter aucune garantie sur le tracé exact des zones car une partie des
+        authorités compétentes fournit des cartes avec une faible résolution. Les données et coordonnées
+        pour une même zone peuvent varier en fonction des sources et de la qualité et précision apportées
+        dans le processus de digitalisation.
+        Finalement, il n&#8217;y a pas de mécanisme automatique pour nous informer des changements apportés
+        par les villes. Les données sur chaque zone sont mises à jour manuellement et dépendent
+        donc des retours d&#8217;utilisateurs comme vous.
+        Pour toutes ces raisons, nous ne pouvons pas garantir la véracité de nos cartes et nous ne
+        sommes pas responsables en cas d&#8217;infractions. Nous vous conseillons fortement de toujours vous
+        fier aux panneaux de signalisation dans les villes concernées."</string>
```
Comment (johnjohndoe) [inline Umweltzone/src/main/res/values-fr/strings.xml:50]: Please remove the wrapping double quotes (`"`).

File: Umweltzone/src/main/res/values-fr/strings.xml [hunk 8]
```diff
@@ -0,0 +1,185 @@
...
+    <string name="appinfo_references_libraries_content">Les bibliothèques d\'application suivantes ont été utilisées dans cette application</string>
+    <string name="appinfo_sourcecode_title">Code source</string>
+    <string name="appinfo_sourcecode_content">Le code source de cette application est disponible en ligne.</string>
+    <string name="appinfo_sourcecode_url_title">Code source de l\'application</string>
+    <string name="appinfo_license_title">Licences</string>
+    <string name="appinfo_license_content">
+        "Le code source de l&#8217;application Umweltzone est sous licence GPLv3.
+        Les coordonnées issues de OpenStreetMap sont sous licence Open Database Licence (ODbL) v1.0.
+        Les autres coordonnées sont sous la licence définie par leur source.
+        Les images et les textes sont sous licence Creative Commons Attribution 3.0 Unported Licence."</string>
```
Comment (johnjohndoe) [inline Umweltzone/src/main/res/values-fr/strings.xml:67]: Please remove the wrapping double quotes (`"`).

File: Umweltzone/src/main/res/values-fr/strings.xml [hunk 9]
```diff
@@ -0,0 +1,185 @@
...
+    <string name="appinfo_license_url_title_gpl">Licence GPLv3</string>
+    <string name="appinfo_license_url_title_creative_commons">Licence Creative Commons</string>
+    <string name="appinfo_build_info_title">Informations de build</string>
+    <string name="appinfo_build_info_time">Temps de build</string>
+    <string name="appinfo_build_info_version_code">Code de version</string>
+    <string name="appinfo_build_info_hash">Hash de build</string>
+    <string name="appinfo_contact_title">Contact</string>
+    <string name="appinfo_contact_content">
+        "Tous les retours, suggestions, idées d&#8217;amélioration et les bugs peuvent être envoyés à
+        l&#8217;adresse suivante : "</string>
```
Comment (johnjohndoe) [inline Umweltzone/src/main/res/values-fr/strings.xml:77]: Please remove the wrapping double quotes (`"`).

File: Umweltzone/src/main/res/values-fr/strings.xml [hunk 10]
```diff
@@ -0,0 +1,185 @@
...
+    <string name="appinfo_rating_url_title">Umweltzone est dans le Google Play Store</string>
+    <string name="city_info_activity_title">Informations sur la ville</string>
+    <string name="city_info_section_title_lez">Détails sur les zones à faibles émissions</string>
+    <string name="city_info_section_title_dpz">Détails sur les zones d\'interdiction des diesels</string>
+    <string name="city_info_section_title_other">Plus d\'informations</string>
+    <string name="city_info_zone_status">gratuit</string>
+    <string name="city_info_zone_number_as_of">
+        "Depuis le <xliff:g example="Oct 1, 2099" id="date">%1$s</xliff:g> seuls
+        les véhicules avec des autocollants de couleur <xliff:g example="with a green" id="with_colors">%2$s</xliff:g>
+        sont autorisés à circuler dans les zones à faibles émissions"</string>
```
Comment (johnjohndoe) [inline Umweltzone/src/main/res/values-fr/strings.xml:89]: Please remove the wrapping double quotes (`"`).

File: Umweltzone/src/main/res/values-fr/strings.xml [hunk 11]
```diff
@@ -0,0 +1,185 @@
...
+    <string name="changelog_version_format">"Version <xliff:g id="version_name">%s</xliff:g>"</string>
+    <string name="city_info_zone_number_since">
+        "Depuis le <xliff:g example="Feb 1, 2016" id="date">%1$s</xliff:g> seuls
+        les véhicules avec un autocollant de couleur <xliff:g example="with a green" id="with_colors">%2$s</xliff:g>
+        sont autorisés à circuler dans les zones à faibles émissions"</string>
```
Comment (johnjohndoe) [inline Umweltzone/src/main/res/values-fr/strings.xml:94]: Please remove the wrapping double quotes (`"`).

File: Umweltzone/src/main/res/values-fr/strings.xml [hunk 12]
```diff
@@ -0,0 +1,185 @@
...
+        les véhicules avec un autocollant de couleur <xliff:g example="with a green" id="with_colors">%2$s</xliff:g>
+        sont autorisés à circuler dans les zones à faibles émissions."</string>
+    <string name="city_info_abroad_licensed_vehicle_zone_info">
+        "Les véhicules immatriculés à l&#8217;étranger sont autorisés à circuler dans la zone à faibles émissions
+        jusqu&#8217;au <xliff:g example="Jan 31, 2016" id="date">%1$s</xliff:g> avec un autocollant de couleur
+        <xliff:g example="with a yellow or a green" id="with_colors">%2$s</xliff:g>"</string>
+    <string name="city_info_list_of_cities">
+        "La zone comprend les villes et les régions suivantes :
+        <xliff:g example="Bochum, Castrop-Rauxel, Essen" id="cities">%1$s</xliff:g> et
+        <xliff:g example="Recklinghausen" id="last_city">%2$s</xliff:g>.     "</string>
```
Comment (johnjohndoe) [inline Umweltzone/src/main/res/values-fr/strings.xml:113]: Please remove the wrapping double quotes (`"`) and the unneeded whitespace. These cause formatting issues.

File: Umweltzone/src/main/res/values-fr/strings.xml [hunk 13]
```diff
@@ -0,0 +1,185 @@
...
+    <string name="city_info_further_information">Plus d&#8217;informations en ligne</string>
+    <string name="city_info_badge_online_title">Formulaire de demande d&#8217;autocollant en ligne</string>
+    <string name="city_info_geometry_source_text">"Source des coordonnées satellite:
+        <xliff:g example="Geoportal Berlin / Umweltzone Berlin" id="geometrySource">%1$s</xliff:g>"</string>
+    <string name="city_info_geometry_updated_at_date_format">dd MM YYYY</string>
+    <string name="city_info_geometry_updated_at_text">Dernière mise à jour du tracé de la zone : <xliff:g example="Sep 6, 2014" id="updatedAt">%1$s</xliff:g></string>
```
Comment (johnjohndoe) [inline Umweltzone/src/main/res/values-fr/strings.xml:119]: Please remove the space before the colon `zone :` -> `zone:`

File: Umweltzone/src/main/res/values-fr/strings.xml [hunk 14]
```diff
@@ -0,0 +1,185 @@
...
+    <string name="city_info_allowed_exhaust_emission_standard_in_diesel_prohibition_zone">
+        "Seuls les véhicules qui respectent les standard d&#8217;émissions Euro <xliff:g example="6" id="number">%1$d</xliff:g>
+        sont autorisés à circuler dans la zone d&#8217;interdiction des diesels."</string>
+    <string name="city_info_zone_number_for_residents_since">
+        "Pour les résidents, ces règles s&#8217;appliquent à partir du <xliff:g example="May 31, 2018" id="date">%1$s</xliff:g>."</string>
+    <string name="city_info_zone_number_for_non_residents_since">
+        "Pour les non-résidents, ces règles s&#8217;appliquent à partir du <xliff:g example="May 31, 2018" id="date">%1$s</xliff:g>."</string>
+    <string name="city_info_provision_applies_to_vehicles">"        Ces règles s&#8217;appliquent aux <xliff:g example="cars and heavy goods vehicles" id="vehicles">%1$s</xliff:g>.     "</string>
```
Comment (johnjohndoe) [inline Umweltzone/src/main/res/values-fr/strings.xml:127]: Please remove the wrapping double quotes (`"`) and the unneeded whitespace. These cause formatting issues.

File: Umweltzone/src/main/res/values-fr/strings.xml [hunk 15]
```diff
@@ -0,0 +1,185 @@
...
+    <string name="zone_not_drawable_notice">"
+        <![CDATA[&#8230; pour que les <b>zones à faibles émissions de %1$s</b> puissent être affichées aussi.<br /><br />
+         Le problème est que %2$s ne publie pas les coordonnées satellites (latitude et longitude)
+         qui décrivent le tracé de la zone dans un <b>format lisible par les ordinateurs</b>.
+         Ces informations ne peuvent pas être extraites d&#8217;une image, d&#8217;un pdf ou d&#8217;une page web.
+         Pour cette raison, il n&#8217;est pas possible de déterminer précisément l&#8217;emplacement et la
+         forme de la zone à faibles émissions. <br /> <br />
+         Mais vous pouvez aider à changer cette situation en <b>envoyant un email</b>.
+         Le message a déjà été écrit pour vous : il ne vous reste qu&#8217;à ajouter votre nom en bas de la page
+         et vous êtes prêt à l&#8217;envoyer.]]>"</string>
```
Comment (johnjohndoe) [inline Umweltzone/src/main/res/values-fr/strings.xml:147]: Please remove the wrapping double quotes (`"`).

File: Umweltzone/src/main/res/values-fr/strings.xml [hunk 16]
```diff
@@ -0,0 +1,185 @@
...
+    <string name="trace_droid_dialog_title">"Envoyer un rapport de plantage &#8230;   "</string>
+    <string name="trace_droid_dialog_message">
+        "Un problème avec l&#8217;application Umweltzone a été détecté. Il serait très utile de nous envoyer
+        un rapport de plantage pour que nous puissions identifier la cause du problème."</string>
```
Comment (johnjohndoe) [inline Umweltzone/src/main/res/values-fr/strings.xml:171]: Please remove the wrapping double quotes (`"`) and the unneeded whitespace. These cause formatting issues.

File: Umweltzone/src/main/res/values-fr/strings.xml [hunk 17]
```diff
@@ -0,0 +1,185 @@
...
+    <string name="snack_engage_beta_testing_title">Utilisateur avancé ?</string>
+    <string name="snack_engage_beta_testing_action">Devenez un beta-testeur</string>
+    <string name="settings_activity_title">Paramètres</string>
+    <string name="settings_category_title_data_privacy">Protection des données</string>
+    <string name="settings_title_google_analytics">Envoyer des statistiques d&#8217;usages anonymisées</string>
+    <string name="settings_summary_google_analytics">
+        "Autoriser la transmission de statistiques d&#8217;usage et des raports de plantages à Google Analytics.
+        Ces données anonymisées sont utilisées par les développeurs de Umweltzone pour analyser les erreurs
+        et bugs afin de continuer à améliorer l&#8217;application."</string>
+    <string name="settings_title_data_privacy_statement_de">"Manifeste de protection des données (en allemand)&#8230;  "</string>
```
Comment (johnjohndoe) [inline Umweltzone/src/main/res/values-fr/strings.xml:184]: Please remove the wrapping double quotes (`"`) and the unneeded whitespace. These cause formatting issues.

screenshot

File: README.md [hunk 18]
```diff
@@ -36,6 +36,7 @@ The app is translated into multiple languages:
 - Polish (FAQs missing)
 - Portuguese
 - Swedish (FAQs missing)
+- French (FAQs missing)
```
Comment (johnjohndoe) [inline README.md:39]: Thanks for adding the language here. Please move it up to have the list in alphabetical order.

**Review by johnjohndoe — CHANGES_REQUESTED** [review]
Hello @laurerunser. Thank you very much for translating to French :+1: 

There are a few strings to be corrected (most are repetitive). Please let me know if you have any questions or need help.

**Comment by laurerunser** [comment]
I did all the changes as you asked ! I'm very sorry for the errors.
Please tell me if this builds. I don't know if this is worth investigating but even before the changes the project was building and running without any errors for me. It also didn't have any formatting problems in the emulator. I have the latest android-studio release and Fedora 29 (but it's probably just my machine).

File: Umweltzone/src/main/res/values-fr/strings.xml (hunk 13 above)
Comment (laurerunser) [reply Umweltzone/src/main/res/values-fr/strings.xml:131]: In French, the convention is that there is a space before and after the colon. Should I still remove it ?

Comment (johnjohndoe) [reply Umweltzone/src/main/res/values-fr/strings.xml:131]: No. That's fine. Thank you for telling me about it.

**Comment by johnjohndoe** [comment]
Nothing to be sorry for. Your contribution is very much appreciated. Please, this is just a normal process of contributing and reviewing it. It does not have to be perfect at the first time - that's why minimum four eyes :eyes: are looking at this.

You can always see if the build succeeds by checking the section at the end:

"All checks have passed" section

Additionally, I always install the app on a real device or emulator, switch the system language to the relevant language and see if all screens work well and look good.

**Review by johnjohndoe — APPROVED** [review]
Merci @laurerunser for the latest update. I reviewed your work and it looks great.

I squashed your commits into a single commit because the later once were corrections of the initial one. I found a single case where the HTML escape has to omitted but that was new to me, too. I fixed that - see `share_intent_message` if you are interested. I added a space before the colon in `city_info_geometry_source_text` to have it formatted consistently as the other colon case.

Again, thank you so much for your contribution!

**Comment by laurerunser** [comment]
Thanks, I'm happy I could help ! I'll try to find some time for the FAQ later to complete the translation.

**Pull request merged**